alembic/versions/2016031317_add_bundle_contents_index_40d61632fd13.py

```diff
@@ -0,0 +1,22 @@
+"""Add bundle contents index
```

Comment:
Can we call this something other than '*_index', which should be reserved for names of indexes in the table?
Maybe bundle_contents_list or bundle_contents_summary?